### PR TITLE
docs: add non-technical OpenClaw install guide from releases

### PR DESCRIPTION
**Pull request by Akane-CN**
## Summary
Add a user-facing quick-start section for non-technical users who want to install OpenClaw from release packages.

### Included
- New README section: **Install OpenClaw from release package (non-technical quick path)**
- English step-by-step flow
- Chinese simplified step-by-step flow
- Command sequence for pairing/install/start/status
- Short troubleshooting note for diagnose artifact handoff

## Why
Current docs are implementation-oriented. This PR adds an end-user path so non-technical users can follow a simple release-download workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +66,39 @@ See `CONTRIBUTING.md` for command examples and required process.
 ## Local workflow notes
 
 - Before making changes, read the skill instructions under `skills/` (especially `skills/pr-review/SKILL.md` and `skills/review-followup/SKILL.md`) to follow repository-specific conventions.
+
+## Install OpenClaw from release package (non-technical quick path)
+
```
Comment (dev01lay2) [inline README.md]: NBS: Consider linking directly to the Releases page (e.g., `[Releases](../../releases)`) instead of saying "Open the Releases page" — saves the user a step.

File: README.md [hunk 2]
```diff
@@ -66,3 +66,39 @@ See `CONTRIBUTING.md` for command examples and required process.
...
+> Status note: this project is still in Phase 1 scaffold, but release packages can already be used as the starting point for local installation flow.
+
+### English
+
+1. Open the **Releases** page of this repository.
+2. Download the ZIP file for your OS/CPU (for example: `linux-x64`, `darwin-arm64`, `windows-x64`).
```
Comment (dev01lay2) [inline README.md]: NBS: The checksum verification step would be more useful with the actual command: `sha256sum -c carrier-*.sha256` (Linux/macOS) or `Get-FileHash` (Windows PowerShell).

**Review by Akane-CN — COMMENTED** [review]
Technical review sweep by @Akane-CN.

CI status: no check data reported from CLI; please verify required checks before merge.

Changed files reviewed:
- `README.md`

- Please ensure edge-case coverage for modified paths and mention test evidence in the PR thread if available.

**Comment by Akane-CN** [comment]
Re-review on current head `396b3d4`.

Checked:
- `README.md`
- `docs/Agent_Installation_Platform_Implementation_Plan.md`
- `docs/Agent_Installation_Platform_PRD.md`

Technical notes:
- `README.md` now defines a clear source-of-truth read order and aligns terminology (`Runtime`, `Diagnose`, memory classes, `P0/P1`) with PRD semantics.
- The implementation plan now explicitly defers scope/priority conflicts to PRD, which reduces spec drift risk.
- PRD includes an explicit conflict-resolution rule plus a terminology reference section, improving consistency for downstream docs and reviews.

CI checks are green on this head (`Daemon Tests`, `Gateway Type Check`, `WIP`). I don’t see blockers in this pass.

**Comment by Akane-CN** [comment]
Addressed the latest suggestions on current head:

- clarified where pairing code comes from (daemon terminal output)
- documented platform-specific daemon executable names (`agentd` / `agentd.exe`)
- added explicit pre-start env note for `OPENAI_API_KEY`
- made diagnose step actionable with exact command: `/diagnose openclaw`
- added direct Releases link and checksum command examples (Linux/macOS + PowerShell)

Please re-check the updated README section.

File: README.md [hunk 3]
```diff
@@ -66,3 +83,53 @@ See `CONTRIBUTING.md` for command examples and required process.
...
+### English
+
+1. Open [**Releases**](../../releases) for this repository.
+2. Download the ZIP package for your OS/CPU (for example: `linux-x64`, `darwin-arm64`, `windows-x64`).
+3. (Optional but recommended) Download the matching `.sha256` file and verify checksum:
+   - Linux/macOS: `sha256sum -c carrier-*.sha256`
+   - Windows PowerShell: `Get-FileHash .\carrier-*.zip -Algorithm SHA256`
+4. Extract the ZIP to a local folder.
+5. Start the daemon from the extracted folder:
+   - macOS/Linux: `agentd`
```
Comment (Keith-CY) [inline README.md]: From the extracted directory, this should be `./agentd` (and `./agentd.exe` or `.\agentd.exe` on Windows shells). Using bare `agentd`/`agentd.exe` assumes PATH contains the current directory, which is not true in default setups.

**Review by Keith-CY — CHANGES_REQUESTED** [review]
Blocking: The quick-start run command currently uses bare executable names. In default shells, this usually fails because the current directory is not on PATH.

NBS: In the checksum step, add a macOS-native command (for example `shasum -a 256`) or note GNU coreutils requirement, since `sha256sum` is not available by default on macOS.

**Comment by Akane-CN** [comment]
Quick follow-up: the two previously blocking doc items are now addressed in the current head (`2f7506f`):
- command invocation uses explicit relative paths (`./agentd` / `.\\agentd.exe`)
- macOS checksum path includes `shasum -a 256`

I’ve enabled squash auto-merge so this can merge automatically once required conditions are satisfied.

**Review by Keith-CY — COMMENTED** [review]
NBS: Link  in README may be fragile from repository root context. Please use a stable path such as  or the full GitHub URL in both English and 中文 sections.

**Review by Keith-CY — COMMENTED** [review]
NBS: README uses ../../releases in markdown links. This can be fragile depending on page context; please use a stable link such as ./releases or the full GitHub releases URL in both English and 中文 sections.

**Review by dev01lay2 — APPROVED** [review]
Re-reviewed current head. Keith-CY confirmed prior CHANGES_REQUESTED was non-blocking only — both blocking items (executable paths, macOS checksum) were fixed in earlier commits. Remaining NBS items (releases link paths) noted but not blocking. LGTM. 🐰

**Review by Keith-CY — APPROVED** [review]
